Sources_v3/Database/DatabaseSession.swift

```diff
@@ -95,6 +95,8 @@ protocol DatabaseSession: UserDatabaseSession, CurrentUserDatabaseSession, Messa
     ) throws -> ChannelDTO
     
     func loadChannel<ExtraData: ExtraDataTypes>(cid: ChannelId) -> ChannelModel<ExtraData>?
+    
+    func delete(_ dto: NSManagedObject)
```

Comment:
Can we please make this a DTO specific. For example: `delete(_ message: MessageDTO)` and move it to `MessageDatabaseSession`? The goal is to have `DatabaseSession` empty and have it acting only as an umbrella protocol for the concrete session protocols.

Reply:
Changed to `delete(message: MessageDTO)`. The parameter name is there to let us call `delete(_ object: NSManagedObject)` from the inside of this method